Irina/src/main/java/collections/Address.java

```diff
@@ -0,0 +1,18 @@
+package collections;
+
+public class Address {
+    public String country;
+    public String city;
+    public String street;
+
+    public Address(String country, String city, String street) {
+        this.country = country;
+        this.city = city;
+        this.street = street;
+    }
+
+    public String getFullAddress(String address) {
+        address = country + ", " + city + ", " + street;
+        return address;
```

Comment:
Could you simplify those 2 lines of code in one single line like: **return country + ...**

Reply:
Done